### PR TITLE
697 error page improvement

### PR DESCRIPTION
**Pull request by hyi**
This PR implemented customized 403 forbidden page exactly as specified by @dtarb. This PR needs to go in for AGU release. @pkdash Please review yourself or assign someone to review as appropriate. @dtarb I can deploy this to a VM for you to test if you'd like. Just let me know if you want me to do so, otherwise, I will not bother to do that since I implemented exactly as shown in the mockup screen capture in the issue.

### CODE REVIEW

File: hs_core/templatetags/hydroshare_tags.py [hunk 1]
```diff
@@ -1,55 +1,14 @@
...
-from mezzanine.core.fields import RichTextField
-from mezzanine.core.forms import get_edit_form
-from mezzanine.utils.cache import nevercache_token, cache_installed
-from mezzanine.utils.html import decode_entities
-from mezzanine.utils.importing import import_dotted_path
-from mezzanine.utils.sites import current_site_id, has_site_permission
-from mezzanine.utils.urls import admin_url
-from mezzanine.utils.views import is_editable
+from future.builtins import int
+
```
Comment (pkdash) [inline hs_core/templatetags/hydroshare_tags.py]: Great cleanup.

File: hs_core/templates/403.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{% extends 'pages/page.html' %}
```
Comment (pkdash) [inline hs_core/templates/403.html]: I am not sure about the content of this page. @dtarb may want to look at this html code or want to test it if gets deployed for testing.

**Comment by pkdash** [comment]
If @dtarb finds the 403 page content to be correct then +1 from me.

**Comment by hyi** [comment]
@dtarb I have deployed this branch to riverdev for you to test 403 forbidden page improvement. It is implemented to mimic your screen capture on the issue. See if it looks good to you. If it looks good, I can then merge this PR.

**Comment by dtarb** [comment]
I am happy with the functionality as it is (so given short of time perhaps just merge).  However if the following enhancement is easy, consider it.  

If I go to the URL of a private resource when not logged in, it gives me the option to log in.  If I log in it takes me to the successful logged in home page.  It would be better if it could remember the resource I was trying to access and attempt to go to that resource landing page (which would open if I have access to it, or give the request permission message again if I do not have access).

**Comment by hyi** [comment]
@dtarb I did some investigation trying to see if there is an easy way to get login success to redirect to the previous resource landing page, and I can confirm this is not so easy to get it done especially given the short time. Specifically, login form and view functions are all done in Mezzanine, and if we want to do this customization, we will have to override the default Mezzanine login view function, which definitely takes some time to debug and get it done right. So I decided to just merge it for AGU. If you want to get this nice-to-have feature (i.e., remember and redirect to the previous resource landing page after logging in) done after AGU, you can create a separate issue to track it. I am merging it now.

**Pull request merged**